configs/sites/tier1/ursa/packages_oneapi.yaml

```diff
@@ -16,22 +16,20 @@ packages:
       modules:
       - intel-oneapi-mpi/2021.13.1
       prefix: /apps/spack-2024-12/linux-rocky9-x86_64/oneapi-2024.2.1/intel-oneapi-mpi-2021.13.1-ss72gbndvat3oz22sa6lhmlbjkeabrn4
-
+  intel-oneapi-mkl:
+    externals:
+    - spec: intel-oneapi-mkl@2024.2.1%oneapi@2024.2.1
+      modules:
+      - intel-oneapi-mkl/2024.2.1
+      prefix: /apps/spack-2024-12/linux-rocky9-x86_64/gcc-11.4.1/intel-oneapi-mkl-2024.2.1-srqwrbzwo2k7hxuuhlrxttburs5jvlat
   intel-oneapi-runtime:
     externals:
     - spec: intel-oneapi-runtime@2024.2.1%oneapi@2024.2.1
       prefix: /apps/spack-2024-12/linux-rocky9-x86_64/gcc-11.4.1/intel-oneapi-compilers-2024.2.1-oqhstbmawnrsdw472p4pjsopj547o6xs/compiler/2024.2
       modules:
       - compiler-rt/2024.2.1
+  gcc-runtime:
+    externals:
+    - spec: gcc-runtime@11.4.1%gcc@11.4.1
+      prefix: /usr
 
-  # If using intel-oneapi-mkl, make appropriate changes below
```

Comment:
All of this is redundant

Reply:
This is what Jessica Meixner tested in https://github.com/ufs-community/ufs-weather-model/pull/2650#issuecomment-2956377120